### DISCUSSION TITLE
Mirror engraver

### DISCUSSION THREAD

**Discussion by Tucacoach** [arkypita/LaserGRBL#2336] [Q&A]
Hi, when i send image to engraver it stay mirrored... whats the problem?

**Comment by Tribo-2dot0** [comment]
ele fica espelhado na tela logo após abri-la? ou grava espelhado?

**Reply by Tucacoach** [reply]
grava espelhado...
estava funcionando "normal". coloquei o rolo giratório e gravou espelhado, mas não mexi em configuração. Não testei novamente sem o rolo giratório

**Reply by Tribo-2dot0** [reply]
Se for apenas o rotativo, gire ele 180º...
Se estiver acontecendo no modo normal, modifique o valor de $3 conforme a tabela, prestando atenção ao eixo que está invertido (X se espelhado horizontalmente, Y se verticalmente).
![Dolar 3](https://github.com/user-attachments/assets/770cc833-2cd5-4fb8-b051-5db7719107c9)

**Reply by Tucacoach** [reply]
Configurei o #3 para 2 e funcionou corretamente com o rolo giratório. Sem o rolo o valor é o 4

**Reply by Tucacoach** [reply]
muito obrigado

**Reply by Tribo-2dot0** [reply]
Você pode deixar o valor sempre em 4 e usar o rolo virado em 180º
